### PR TITLE
[IS-851] - Add Navarchos to public Prow

### PR DESCRIPTION
**Pull request by theobarberbany**
This PR adds Navarchos to the public Prow instance.

### CODE REVIEW

File: config/jobs/navarchos/navarchos-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+container_template_large: &container_template_large
+  <<: *container_template
+  resources:
+    requests:
+      cpu: 2
+      memory: 4Gi
+    limits:
+      cpu: 4
+      memory: 8Gi
```
Comment (gargath) [inline config/jobs/navarchos/navarchos-presubmits.yaml]: The nodes we're using only have 7.2Gi of RAM available, so this limit will never be reached.

**Review by wonderhoss — COMMENTED** [review]
LGTM, apart from the container memory limit.

File: config/jobs/navarchos/navarchos-presubmits.yaml (hunk 1 above)
Comment (theobarberbany) [reply config/jobs/navarchos/navarchos-presubmits.yaml]: Ok, I'll change it to `6Gi`

**Review by wonderhoss — APPROVED** [review]
LGTM

**Pull request merged**